mycodo/utils/method.py

```diff
@@ -1,115 +1,94 @@
 # coding=utf-8
 import datetime
+import time
 from math import sin, radians
 
-from mycodo.config import SQL_DATABASE_MYCODO
-from mycodo.databases.utils import session_scope
+from mycodo.utils.system_pi import get_sec
 from mycodo.utils.database import db_retrieve_table_daemon
+from mycodo.databases.models import Method
+from mycodo.databases.models import MethodData
 
-MYCODO_DB_PATH = 'sqlite:///' + SQL_DATABASE_MYCODO
```

Comment:
db write access completly removed from this file. it was accessing controller tables that are now written within the controllers themselves.